src/main/java/de/craften/plugins/rpgplus/scripting/api/inventory/InventoryModule.java

```diff
@@ -77,6 +84,36 @@ public Varargs takeItems(Varargs args) {
         return LuaValue.varargsOf(missingAmounts);
     }
 
+    @LuaFunction("setItem")
+    public Varargs setItem(Varargs args) {
+        Player player = ScriptUtil.getPlayer(args.arg(1));
+        ItemMatcher matcher = ScriptUtil.createItemMatcher(args.arg(3));
+        if (args.isnumber(2)) {
+            int slot = args.toint(2);
+            player.getInventory().setItem(slot, matcher.getItemStack());
+        } else {
+        	String s = args.tojstring(2);
+        	switch (s) {
```

Comment:
Also, these "magic" values (chestplate, leggings, ...) should be documented in Luadoc. Maybe add another magic "hand" value that sets the item in the current inventory slot (swapping the currently selected item)? That would be cool!